bundle/src/test/java/com/adobe/acs/commons/genericlists/impl/GenericListAdapterFactoryTest.java

```diff
@@ -130,11 +130,6 @@ protected GenericList adaptToGenericList(Page page) {
     @Test
     public void test_that_adapting_page_with_null_template_returns_null() {
         Page wrongPage = mock(Page.class);
-        when(wrongPage.getProperties()).thenAnswer(new Answer<ValueMap>() {
```

Comment:
TBH I'm not sure if this test is useful at all anymore; when I rewrote the generic list feature I implemented it as sling models.  Therefore, it's feasible that adaptTo would always return non-null as long as the resource itself isn't null.

Reply:
Mockito 2 is quite picky and complains about unused mockings. So this means that ```wrongPage.getProperties()``` is not called in the test execution.